charts/consul/templates/mesh-gateway-deployment.yaml

```
@@ -337,7 +341,7 @@ spec:
           lifecycle:
             preStop:
               exec:
-                command: ["/bin/sh", "-ec", "/consul-bin/consul services deregister -id=\"{{ .Values.meshGateway.consulServiceName }}\""]
+                command: ["/bin/sh", "-ec", "/consul-bin/consul logout -token-file=/consul/login/acl-token"]
```

I believe the default behaviour is to try to use `/consul/login/acl-token` ?
```suggestion
                command: ["/bin/sh", "-ec", "/consul-bin/consul logout"]
```